### PR TITLE
add bls PublicKey GetPoint

### PR DESCRIPTION
**Pull request by lourkeur**
Simply an accessor. This would help with the DKG for F3B-IBE.

### CODE REVIEW

**Comment by pierluca** [comment]
This seems to break the PublicKey abstraction, which arguably is only useful to hide underlying the kyber.Point.
Can you clarify why you need this, the problem you're trying to solve, etc. ?

File: crypto/bls/bls_test.go [hunk 1]
```diff
@@ -88,6 +88,13 @@ func TestPublicKey_MarshalText(t *testing.T) {
 	require.EqualError(t, err, fake.Err("couldn't marshal"))
 }
 
+func TestPublicKey_GetPoint(t *testing.T) {
+	point := suite.Point()
+	pk := PublicKey{point: point}
```
Comment (jbsv) [inline crypto/bls/bls_test.go:93]:
```suggestion
	pk := NewPublicKeyFromPoint(point)
```

Comment (jbsv) [reply crypto/bls/bls_test.go:93]: Should be the same test as [for the ed25519 implementation](https://github.com/dedis/dela/blob/6cd3d62903806f7863ff29a75c12c6c6bfe06085/crypto/ed25519/ed25519_test.go#L38), in which case the GetPoint() is needed.

**Review by jbsv — CHANGES_REQUESTED** [review]
align test with ed25519

**Comment by pierluca** [comment]
Thank you @lourkeur for the clarification.
That makes sense 👍 
This works for me, the failing test is unrelated to this PR.

**Review by jbsv — APPROVED** [review]

**Pull request merged**